### ISSUE TITLE
Fix inconsistency in array style arguements supplied to operations API

### ISSUE THREAD

**Issue by tplooker** [decentralized-identity/bbs-signature#113]
Currently sign and verify function signatures look like the following

```
signature = Sign(SK, PK, (msg_1,..., msg_L), (H_1,..., H_L))
```

and

```
result = Verify(PK, (msg_1,..., msg_L), (H_1,..., H_L), signature)
```

Whereas the spkgen operations function signature looks like

```
spk = SpkGen(PK, (msg_1,..., msg_L), (H_1,..., H_L), RevealedIndexes, signature, pm)
```

where RevealedIndexes is a vector/array.

We need to either change the RevealedIndexes to be more like the the destructed syntax for messages and generators e.g `(msg_1,..., msg_L), (H_1,..., H_L)` or convert all array style arguments to inputs like the current definition for RevealedIndexes.

**Comment by BasileiosKal** [comment]
My vote is for the second option, i.e., convert `RevealedIndexes` to something like `(i1, i2, ..., iR)`. My reasoning is that otherwise, we will have to add many steps in each algorithm to convert each input to its destructed form (i.e., `1.  (msg_1, ..., msg_L) = messages`, `2.  (H_1, ..., H_L) = generators` etc.) that may heart readability.

On the other hand, i feel like the term RevealedIndexes is very easy to understand, so although not internally consistent, leaving the inputs as they are (and sacrificing consistency) may be the best tradeoff for readability.

**Comment by tplooker** [comment]
Being addressed by #138
